package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "node-js-course",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "babel src -d dist",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./dist/app.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sasha-klimashevich/node-js-course.git"
+  },
+  "author": "Aliaksandra Klimashevich",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/sasha-klimashevich/node-js-course/issues"
+  },
+  "homepage": "https://github.com/sasha-klimashevich/node-js-course#readme",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
```

Comment:
Remove all `^`, please

FYI: The `npm` creates `package-lock.json`. But if user uses `npm` less than 5 version. The `^` effects to download up-to-date from resource. It might be problem for running application.
[using-a-package](https://docs.npmjs.com/getting-started/using-a-package.json)

Reply:
Done, thx